Client/Helpers/FxALoginHelper.swift

```diff
@@ -54,15 +54,21 @@ class FxALoginHelper {
     // to changing of user settings and push notifications.
     func application(_ application: UIApplication, didLoadProfile profile: Profile) {
         self.profile = profile
+        self.account = profile.getAccount()
 
-        guard let account = profile.getAccount() else {
+        guard let account = self.account else {
             // There's no account, no further action.
             return loginDidFail()
```

Comment:
What does loginDidFail do? Account is also `nil` if the user simply never logged in to FxA.